opentelemetry-sdk/src/logs/record.rs

```diff
@@ -0,0 +1,203 @@
+use opentelemetry::{
+    logs::{AnyValue, LogRecord, Severity},
+    trace::{SpanContext, SpanId, TraceFlags, TraceId},
+    Key,
+};
+use std::{borrow::Cow, time::SystemTime};
+
+#[derive(Debug, Clone, Default)]
+#[non_exhaustive]
+/// LogRecord represents all data carried by a log record, and
+/// is provided to `LogExporter`s as input.
+pub struct SdkLogRecord {
```

Comment:
We have be overriding names like `TracerProvider` between API and SDK for a while. I think it we should be consistent here. Maybe just `LogRecord`? Users can rename as needed when importing

Reply:
Yes, renamed it to `LogRecord` now.